nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -498,8 +499,17 @@ def transcribe(
             A list of transcriptions (or raw log probabilities if logprobs is True) in the same order 
             as paths2audio_files
         """
-        if timestamps:
-            raise NotImplementedError("Computing timestamps are not supported for this model yet.")
+        if timestamps is not None and timestamps:
+            # TODO: Handle this key gracefully later
+
+            if timestamps is True:
+                timestamps = 'yes'
+            elif timestamps is False:
+                timestamps = 'no'
+            else:
+                timestamps = str(timestamps)
+                assert timestamps in ('yes', 'no', 'timestamp', 'notimestamp')
```

Comment:
```suggestion
                assert timestamps in ('yes', 'no', 'timestamp', 'notimestamp', '1', '0')
```
are also allowed by the prompt format IIRC